public/src/main/java/com/revenuecat/purchases/EntitlementInfo.kt

```diff
@@ -106,6 +107,7 @@ data class EntitlementInfo(
         get() = jsonObject
 
     /** @suppress */
+    @OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
```

Comment:
This is not transitive right? meaning that calling `toString` doesn't require to opt in as well

Reply:
That's correct. If you use `@OptIn`, it doesn't propagate. It would if I just annotated with the `@ ExperimentalPreviewRevenueCatPurchasesAPI` again